stl/inc/format

```diff
@@ -2407,16 +2431,10 @@ _NODISCARD _OutputIt _Fmt_write(
                     }
                 }
             }
+            break;
+        default:

```

Comment:
`default` is unnecessary.

Reply:
A few lines above we have:
```cpp
 switch (_Specs._Type) {
    case '\0':
        break;
    case 'A':
        _To_upper = true;
        [[fallthrough]];
    case 'a':
        _Format   = chars_format::hex;
        _Exponent = 'p';
        break;
```

So I believe this is the right choice